web_src/js/markup/anchors.js

```diff
@@ -30,6 +30,20 @@ export function initMarkupAnchors() {
     });
     heading.prepend(a);
   }
+  for (const anchor of document.querySelectorAll('.markup [dir="auto"] > a[href^="#"][rel="nofollow"]')) {
```

Comment:
```suggestion
  for (const anchor of document.querySelectorAll('.markup a[href^="#"]')) {
```

Seems good enough and will catch nested links as well.